testing/libpuzzle/APKBUILD

```diff
@@ -0,0 +1,53 @@
+# Developer: Frank DENIS <j at pureftpd.org>
+# Maintainer: Timofey Bychkov <boobooin@gmail.com>
+pkgname=libpuzzle
+subpackages="$pkgname-doc"
+pkgver=0.11
+pkgrel=0
+pkgdesc="The Puzzle library is designed to find visually similar images."
+url="http://libpuzzle.pureftpd.org/project/libpuzzle"
+arch="all"
+license="BSD"
```

Comment:
The puzzle library is free. It is released under the 2-clauses BSD public license

So according spdx it needs change